src/alfresco-core-rest-api/src/ApiClient.js

```diff
@@ -54,7 +54,7 @@
      * @type {Number}
      * @default 60000
      */
-    this.timeout = 60000;
+    this.timeout = undefined;
```

Comment:
Maybe we disable only "deadline" one and leave the other as it is?
The api supports setting it like

```
.timeout({
    response: 5000,  // Wait 5 seconds for the server to start sending,
    deadline: 60000, // but allow 1 minute for the file to finish loading.
  })
```

You can see the docs here: https://visionmedia.github.io/superagent/

Reply:
OK

Reply:
yes this is the point where we will add also the upload timer